bouachain/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "bouachain",
+  "assets": [
+    {
+      "description": "The native token of the Bouachain blockchain, used for transaction fees, staking, and governance.",
+      "denom_units": [
+        {
+          "denom": "uBOUA",
+          "exponent": 0
+        }
+	],      
+      "base": "bouacoin",
+      "name": "BOUACOIN",
+      "display": "BOUA",
```

Comment:
This display denom is not defined under denom_units, but must be. Usually this is used to inform the number of decimals/exponents.